proto_bin.h

```diff
@@ -0,0 +1,9 @@
+#ifndef PROTO_BIN_H
+#define PROTO_BIN_H
+
+void complete_nread_binary(conn *c);
+int try_read_command_binary(conn *c);
```

Comment:
```suggestion
int try_read_command_binary(conn *c);
void complete_nread_binary(conn *c);
```
순서를 ascii와 동일하게 맞추는 것은 어떤가요?